fdbrpc/FlowTransport.actor.cpp

```diff
@@ -889,7 +901,14 @@ static void scanPackets(TransportData* transport,
 		}
 
 		ASSERT(!reader.empty());
-		deliver(transport, Endpoint({ peerAddress }, token), std::move(reader), true);
+		TaskPriority priority = transport->endpoints.getPriority(token);
+		// we ignore packets to unknown endpoints if they're not going to a stream anyways, so we can just
+		// return here. The main place where this seems to happen is if a ReplyPromise is not waited on
+		// long enough.
+		// It would be slightly more elegant to put this if-block
```

Comment:
Was there more to this thought?

Reply:
haha, yes. This is my squirrel-mind doing many things at once